File: android/versioned-abis/expoview-abi31_0_0/src/main/java/abi31_0_0/expo/modules/sms/SMSPackage.java

package abi31_0_0.expo.modules.sms;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi31_0_0.expo.core.BasePackage;
import abi31_0_0.expo.core.ExportedModule;
import abi31_0_0.expo.core.interfaces.InternalModule;

public class SMSPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context reactContext) {
    return Collections.singletonList((ExportedModule) new SMSModule(reactContext));
  }
}

File: android/versioned-abis/expoview-abi47_0_0/src/main/java/abi47_0_0/expo/modules/sms/SMSPackage.kt
package abi47_0_0.expo.modules.sms

import android.content.Context
import abi47_0_0.expo.modules.core.BasePackage
import abi47_0_0.expo.modules.core.ExportedModule

class SMSPackage : BasePackage() {
  override fun createExportedModules(reactContext: Context): List<ExportedModule> {
    return listOf(SMSModule(reactContext))
  }
}
